### DISCUSSION TITLE
CUDA version change

### DISCUSSION THREAD

**Discussion by NiklasGustafsson** [xamarin/TorchSharp#825] [Announcements]
With an update to libtorch 1.13, the underlying CUDA version is chainging to 11.7 instead of 11.3. This will mean that any project references to libtorch-cuda-11.3-* will have to change, and that any CUDA drivers you use have to be compatible with 11.7.

The new TorchSharp-cuda packages will reference the right version of libtorch-cuda automatically.

**Comment by NiklasGustafsson** [comment]
This new release of TorchSharp has been made as v0.99.0.

**Comment by TqJunglete** [comment]
> With an update to libtorch 1.13, the underlying CUDA version is chainging to 11.7 instead of 11.3. This will mean that any project references to libtorch-cuda-11.3-* will have to change, and that any CUDA drivers you use have to be compatible with 11.7.
> 
> 
> 
> The new TorchSharp-cuda packages will reference the right version of libtorch-cuda automatically.

**Reply by NiklasGustafsson** [reply]
@TqJunglete -- did you have a question about the change?
